### PR TITLE
Integrate monitoring into piman.py

### PR DESCRIPTION
**Pull request by EveretteLi**
> enable "sudo python3 build/piman.pyz monitoring"
> put monitoring.config, host.csv into build folder in order to run monitoring from piman.pyz
> change the name of monitoring-client.py into monitoring_client to support the import in piman.py
> change piman.py. Start thread for monitoring on command "monitoring"
> check piman_README.md for more details

### CODE REVIEW

File: make_zipapp.sh [hunk 1]
```diff
@@ -8,6 +8,8 @@ mkdir build/piman.app
     mv $(ls | grep -v -) ../../../piman.app
 )
 cp -r logging.conf utility tftp dhcp monitoring tcp *.py build/piman.app
+# copy hosts.csv and monitoring.config into build
+cp -r hosts.csv monitoring/monitoring.config build
```
Comment (breed) [inline make_zipapp.sh]: hosts.csv should not go into build

File: piman_README.md [hunk 2]
```diff
@@ -128,6 +128,23 @@ The official implementation is specified as `mapper(switch_address,interface, po
 
 **Get Mac address of given port**
 
+#### Starting monitoring on rasberry pis
+``` python
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
+    monitor_thread.start()
```
Comment (breed) [inline piman_README.md]: you don't need to do the thread if you are just going to start and join

File: piman.py [hunk 3]
```diff
@@ -106,6 +108,11 @@ def server():
     # when it is run, so after it is closed by a keyboard interrupt, the above
     # lines run closing the rest of the threads. ntp_thread will already be stopped
 
+# start monitoring thread
+def monitoring():
+    monitor_thread= Thread(target = monitoring_client.start_from_piman, name= "monitoring")
```
Comment (breed) [inline piman.py]: you don't need a thread if you are just going to start and join

File: monitoring/monitoring_client.py [hunk 4]
```diff
@@ -0,0 +1,115 @@
+"""
```
Comment (breed) [inline monitoring/monitoring_client.py:1]: i think this is supposed to be a rename. note that there are changes to this file.

**Review by breed — CHANGES_REQUESTED** [review]

**Comment by EveretteLi** [comment]
The related changes have been made.

Thanks!

**Pull request closed without merging**